package.json

```diff
@@ -172,7 +172,8 @@
     "workbox-webpack-plugin": "^6.0.2"
   },
   "engines": {
-    "node": ">=12 <15"
+      "node": "14.2.0",
+      "npm": "6.14.13"
```

Comment:
Do we need to hardcode these? The project should work fine with node v12+

Reply:
Probably not, it's something I was experimenting with on a previous attempt but they should make no difference, let me amend that.